packages/rich-text/src/to-plain-text.js

```diff
@@ -0,0 +1,17 @@
+const { DOMParser } = window;
+
+/**
+ * Removes any HTML tags from the provided string.
+ *
+ * @param {string} html The string containing html.
+ *
+ * @return {string} The text content with any html removed.
+ */
+export function toPlainText( html ) {
+	if ( ! html ) {
+		return '';
+	}
+
+	const document = new DOMParser().parseFromString( html, 'text/html' );
```

Comment:
Could you use the existing rich text functions? `create( { html } ).text`.

Reply:
Good point, I've switch the implementation to that. 👍